nautilus_trader/adapters/interactive_brokers/client/client.py

```diff
@@ -661,7 +662,7 @@ async def _process_message(self, msg: str) -> bool:
         # order, process real-time ticks, etc.) and then calls the corresponding
         # method from the EWrapper. Many of those methods are overridden in the client
         # manager and handler classes to support custom processing required for Nautilus.
-        await asyncio.to_thread(self._eclient.decoder.interpret, fields)
+        await asyncio.to_thread(self._eclient.decoder.interpret, fields, 0)
```

Comment:
This 0 here is certainly wrong, but not sure how to get the message Id. 

Some hints here: https://github.com/nautechsystems/nautilus_ibapi/blob/main/ibapi/client.py#L426